test/plugins/test_super_check.py

```diff
@@ -56,9 +56,7 @@ def {method}(self):         #=A
         """
     ).format(method=method)
     messages = run_pylint(source, MSG_IDS)
-    expected = {
-        "A:super-method-not-called:super(...).setUpTestData() not called (django.test.testcases.TestCase)"
-    }
+    expected = set()
```

Comment:
Couldn't find changelog for this fix. Updated according to the error trace in the test failure.